src/fhirdefs/load.ts

```diff
@@ -257,7 +264,7 @@ export function loadCustomResources(resourceDir: string, defs: FHIRDefinitions):
         // All resources are added to the predefined map, so that this map can later be used to
         // access predefined resources in the IG Exporter
         defs.addPredefinedResource(file, resourceJSON);
-        if (pathEnd !== 'examples') {
+        if (!dirPath.endsWith('examples')) {
```

Comment:
I think that using [path.basename](https://nodejs.org/api/path.html#pathbasenamepath-ext) instead of endsWith would be closer to the original intent (and wouldn't match on things like `input/definitelynotexamples`).

Reply:
Changed in commit a080a34! That should make this more robust.